pkg/log/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/keikoproj/upgrade-manager/pkg/log
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.25.0
+	github.com/aws/aws-sdk-go v1.25.42
```

Comment:
This was intentionally rolled back to v1.25.0 in https://github.com/keikoproj/upgrade-manager/issues/30. Newer versions of the SDK use an updated mechanism of getting the aws creds and require some changes to the AMI before doing that.

Has something changed?